tests/units/test_tokens.py

```diff
@@ -193,3 +193,14 @@ def test_create_token_hit(setup_db, token_type, hit_info):
         src_ip=src_ip,
         hit_info=hit_info,
     )
+
+
+@pytest.mark.parametrize(
+    "in_url, expected_out",
+    [("JBSWY3DPGIQQ.G01.yh6wfyh752qi06e35f9b0260f.127.0.0.1", b"Hello2!")],
```

Comment:
test are free add a few more. add ones that have `.` in them and I see we handle lower case add that in as well.